test/junit/scala/collection/SortedSetTest.scala

```diff
@@ -55,6 +55,15 @@ class SortedSetTest {
       assert(count > 10)
     }
   }
+
+  @Test
+  def min_max_differentOrdering(): Unit = {
+    val set = SortedSet(1, 2, 3)(Ordering[Int].reverse)
```

Comment:
It isn't obvious that this is testing the fallthrough to super instead of the `isReverseOf` line.

Reply:
I... actually missed that. You're right, it's hitting `isReverseOf`. I'll change it to do a `fromLessThan` or something